src/components/OpenSource/ContributionsCopy.js

```diff
@@ -0,0 +1,21 @@
+import React, { Fragment } from 'react'
+import { SectionTitle } from '../Typography'
+const ContributionsCopy = props => {
+  const {
+    openSourceMetaRepoCount: projects,
+    openSourceMetaPullRequestCount: contributions,
+    contributionsSectionTitleLine1: first,
+    contributionsSectionTitleLine2: second,
+    contributionsSectionTitleLine3: third
+  } = props
+
+  return (
+    <Fragment>
+      <SectionTitle reverse>
+        {first} {contributions} {second} {projects} {third}
```

Comment:
I assume that it was intentional for the line breaks to correspond with the numbers?

Designs:
![image](https://user-images.githubusercontent.com/15656538/58711937-79f71300-83b7-11e9-8516-91355d849cb7.png)

This branch:
![image](https://user-images.githubusercontent.com/15656538/58711985-8c714c80-83b7-11e9-937d-be4168609635.png)

Reply:
Have added but what about situations like this 
![image](https://user-images.githubusercontent.com/7108120/58721152-1b885f80-83cc-11e9-9706-f50d68b426f4.png)